package.json

```diff
@@ -1,37 +1,46 @@
 {
-  "name": "@esri/hub-ckeditor5-custom-build",
+  "name": "ckeditor5-custom-build",
   "author": "CKSource",
   "description": "A custom CKEditor 5 build made by the CKEditor 5 online builder.",
-  "version": "0.0.4",
+  "version": "0.0.1",
   "license": "SEE LICENSE IN LICENSE.md",
+  "private": true,
   "main": "./build/ckeditor.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/esridc/ckeditor5-custom-build.git"
+  "types": "./build/ckeditor.d.ts",
+  "dependencies": {
+    "@ckeditor/ckeditor5-alignment": "40.1.0",
```

Comment:
These deps were in `devDependencies` before, do they need to be in `dependencies`? This may be OK if this was copied/pasted directly from CKEditor; I don't recall if I manually moved these to `devDependencies` or if they were already there when I set up this repo.